etc/tests/test_bootstrapper.py

```diff
@@ -542,7 +542,7 @@ def test_package_installation_with_target_path(monkeypatch, virtualenv):
 
     virtualenv.run("python3 -m pip install --target='/tmp/lib/' bleach==3.1.5")
     virtualenv.run("python3 -m pip install --target='/tmp/lib/' ansiwrap==0.7.0")
-    virtualenv.run("python3 -m pip install --target='/tmp/lib/' packaging==20.4")
+    virtualenv.run("python3 -m pip install --target='/tmp/lib/' packaging==20.7")
```

Comment:
Was TS this yesterday. Although we explicitly install 20.4 into our test virtualenv, Installation of the bleach package ultimately pulls in 20.7. I cant seem to pinpoint where (bleach has an unpinned req on packaging). Since packaging will pull this in regardless we can probably remove this line. Otherwise lgtm